src/main/scala/org/broadinstitute/dsde/agora/server/webservice/configurations/ConfigurationsService.scala

```diff
@@ -0,0 +1,71 @@
+
+package org.broadinstitute.dsde.agora.server.webservice.configurations
+
+import com.wordnik.swagger.annotations._
+import org.broadinstitute.dsde.agora.server.model.AgoraEntity
+import org.broadinstitute.dsde.agora.server.webservice.{ServiceHandlerProps, AgoraService}
+import org.broadinstitute.dsde.agora.server.webservice.util.ApiUtil
+
+trait ConfigurationsService extends AgoraService {
+  this: ServiceHandlerProps =>
+  // Require a concrete ServiceHandlerProps creator to be mixed in
+
+  def path = ApiUtil.Configurations.path
+
+  @ApiOperation(value = "Get a task configuration in the task configuration repository matching namespace, name, and snapshot id",
+    nickname = "configurations",
+    httpMethod = "GET",
+    produces = "application/json",
+    response = classOf[AgoraEntity],
+    notes = "API is rapidly changing.")
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "namespace", required = true, dataType = "string", paramType = "path", value = "Namespace"),
+    new ApiImplicitParam(name = "name", required = true, dataType = "string", paramType = "path", value = "Name"),
+    new ApiImplicitParam(name = "snapshotId", required = true, dataType = "string", paramType = "path", value = "SnapshotId")
+  ))
+  @ApiResponses(Array(
+    new ApiResponse(code = 200, message = "Successful Request"),
+    new ApiResponse(code = 404, message = "Task Configuration Not Found"),
+    new ApiResponse(code = 500, message = "Internal Error")
+  ))
+  override def queryByNamespaceNameSnapshotIdRoute = super.queryByNamespaceNameSnapshotIdRoute
```

Comment:
exactly... not ideal but swagger doesn't give us a choice. If we could have non static annotation strings we wouldn't need to do this.